hw/ip/clkmgr/dv/clkmgr_sim_cfg.hjson

```diff
@@ -55,7 +55,9 @@
   uvm_test_seq: clkmgr_base_vseq
 
   // Disable clearing interrupts since clkmgr doesn't have any.
-  run_opts: ["+do_clear_all_interrupts=0"]
+  run_opts: ["+do_clear_all_interrupts=0",
```

Comment:
where this plusarg is used?

Reply:
In cip_base: 
https://cs.opensource.google/opentitan/opentitan/+/master:hw/dv/sv/cip_lib/seq_lib/cip_base_vseq.sv;drc=53874025b840788d5e66ffddae8fe7840a9cf4df;l=144